src/base/server_type.cpp

```diff
@@ -104,6 +107,37 @@ bool is_tran_server_with_remote_storage ()
   return false;
 }
 
+void set_server_type_from_arg (int argc, char **argv)
+{
+  if (argc < 4)
+    {
+      return;
+    }
+
+  int arg = 0;
```

Comment:
Bool if variable is a flag.
```suggestion
  bool server_type_mnemonic_found = false;
```